src/pivotal-ui/components/header/header.scss

```diff
@@ -143,14 +143,19 @@ header ul li {
   padding: 0;
   margin: 0;
 }
+
 header a {
   color: $header-subtle-text-color;
+  &:focus {
+    outline: none;
```

Comment:
Oh bother. Wasn't thinking that through fully when pairing with @jerrygabra. That may have to go back in.